apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep1.vue

```diff
@@ -0,0 +1,226 @@
+                                                                  class="beta"
+                                            >{{ $t('beta') }}</span>
+                                        </p>
+                                        <div class="plugin-description">
+                                            <span class="plugin-description-text">
+                                                {{ item.tags.description }}
+                                            </span><p-anchor size="sm"
+                                                             :highlight="true"
+                                            >
+                                                {{ $t('learn more') }}
+                                            </p-anchor>
+                                        </div>
+                                        <div v-if="item.labels">
+                                            <p-label v-for="(label, idx) in item.labels"
+                                                     :key="`${label}-${idx}`"
+                                                     class="mr-2 mb-2"
+                                                     :text="label"
+                                            />
+                                        </div>
+                                    </div>
+                                </div>
+                                <p-button style-type="secondary"
+                                          class="create-button"
+                                >
+                                    {{ $t('Select') }}
+                                </p-button>
+                            </div>
+                        </template>
+                    </p-board-item>
+                </div>
+            </p-data-loader>
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { reactive } from 'vue';
+
+import {
+    PSearch, PDataLoader, PBoardItem, PLazyImg, PButton, PLabel, PAnchor,
+} from '@spaceone/design-system';
+import { get } from 'lodash';
+
+import { getPageStart } from '@cloudforet/core-lib/component-util/pagination';
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+import { ApiQueryHelper } from '@cloudforet/core-lib/space-connector/helper';
+
+import { assetUrlConverter } from '@/lib/helper/asset-helper';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+import { BACKGROUND_COLOR } from '@/styles/colorsets';
+
+import Step1SearchFilter from '@/services/asset-inventory/collector/create-collector/modules/Step1SearchFilter.vue';
+
+const state = reactive({
+    searchValue: '',
+    pluginList: [] as any[],
+    loading: false,
+});
+
+const pluginApiQuery = new ApiQueryHelper();
+const getPlugins = async () => {
+    state.loading = true;
+    try {
+        pluginApiQuery.setPage(getPageStart(1, 10), 10).setSort('name', false)
+            .setFilters([{ v: state.searchValue }]);
```

Comment:
suggestion: I recommend you to use `useQueryTags` for filters since this feature in `QueryHelper` is gonna be deprecated! 🥲
But it is not important to this feature, so please improve your code after implementing the feature first 😁

Reply:
I didn't know that! Thank you for the comments!